plugins/modules/ios_vrf_global.py

```diff
@@ -0,0 +1,677 @@
+          vpn:
+            description: Configure vpn-id for the VRF as specified in RFC 2685.
+            type: dict
+            suboptions:
+              id:
+                description: Configure vpn-id in RFC 2685 format.
+                type: str
+  running_config:
+    description:
+      - This option is used only with state I(parsed).
+      - The value of this option should be the output received from the IOS device by
+        executing the command B(show running-config | section ^vrf).
+      - The state I(parsed) reads the configuration from C(running_config) option and
+        transforms it into Ansible structured data as per the resource module's argspec
+        and the value is then returned in the I(parsed) key within the result.
+    type: str
+  state:
+    choices: [parsed, gathered, deleted, merged, replaced, rendered, overridden, purged]
+    default: merged
+    description:
+      - The state the configuration should be left in
+      - The states I(rendered), I(gathered) and I(parsed) does not perform any change
+        on the device.
+      - The state I(rendered) will transform the configuration in C(config) option to
+        platform specific CLI commands which will be returned in the I(rendered) key
+        within the result. For state I(rendered) active connection to remote host is
+        not required.
+      - The state I(gathered) will fetch the running configuration from device and transform
+        it into structured data in the format as per the resource module argspec and
+        the value is returned in the I(gathered) key within the result.
+      - The state I(parsed) reads the configuration from C(running_config) option and
+        transforms it into JSON format as per the resource module parameters and the
+        value is returned in the I(parsed) key within the result. The value of C(running_config)
+        option should be the same format as the output of command I(show running-config | section vrf).
+        connection to remote host is not required.
+    type: str
+"""
+
+EXAMPLES = """
+# Using merged
+# Before state:
+# -------------
+# admin#show running-config | section ^vrf
+#
+- name: Merge provided configuration with device configuration
+  hosts: ios
+  gather_facts: false
+  tasks:
+    - name: Merge provided configuration with device configuration
+      cisco.ios.ios_vrf_global:
+        config:
+          vrfs:
+            - name: VRF2
+              description: This is a test VRF for merged state
+              ipv4:
+                multicast:
+                  multitopology: true
+              ipv6:
+                multicast:
+                  multitopology: true
+              rd: "2:3"
+              route_target:
+                export: "192.0.2.0:100"
+                import_config: "192.0.2.3:200"
+              vpn:
+                id: "2:45"
+              vnet:
+                tag: 200
+        state: merged
+# Task output
+# -------------
+# commands:
```

Comment:
we add before and after in example as a practice we follow. 
use yaml call back to add the task output here.

Reference - https://github.com/ansible-collections/cisco.ios/blob/main/plugins/modules/ios_bgp_global.py